Folder/email_notifier.py

```diff
@@ -0,0 +1,32 @@
+import os
+import smtplib
+import requests
+from dotenv import load_dotenv
+
+
+class EmailNotifier:
+
+    def __init__(self):
+        self._smtp_username = os.environ["SMTP_USERNAME"]
+        self._smtp_password = os.environ["SMTP_PASSWORD"]
+        self._sheety_users_endpoint = os.environ["SHEETY_USERS_EP"]
+        self.sheety_header = {"Authorization": os.environ["SHEETY_BEARER"]}
+        self.users_data = []
+
+    def get_emails_data(self):
+
+        users_response = requests.get(self._sheety_users_endpoint, headers=self.sheety_header)
+        users_response.raise_for_status()
+        self.users_data = users_response.json()["users"]
+        return self.users_data
+
+    def send_emails(self, user_email, message_body):
+
+        with smtplib.SMTP("smtp.gmail.com", port=587) as connection:
+            connection.starttls()
+            connection.login(user=self._smtp_username, password=self._smtp_password)
+            connection.sendmail(
```

Comment:
you might wanna check if it sends the email correctly. Does this function return an error? You can add a print / log statement at the end, saying it was succesful

Reply:
True, worked when I was testing but good idea to have one just to return a confirmation when successful

Reply:
yea if I wrote this, after `sendmail` i would do
```
print(f"Sent email to {user_email}")
```

Reply:
I added an exception but it's pretty extra compared to that heh, probably didn't need to but wanted to use what I've been learning